control/grpc.py

```diff
@@ -91,7 +116,7 @@ def _alloc_cluster(self) -> str:
         )
         return name
 
-    def create_bdev(self, request, context=None):
+    def create_bdev_safe(self, request, context=None):
```

Comment:
Why renaming? are we planning to allow unsafe calls?

Reply:
@oritwas we have some cases in which we call one RPC function from within another one. Right now we call get_subsystems and remove_namespace from delete_bdev as there was a request to automatically remove namespaces using the bdev when this bdev is deleted. So, for now I left all the duplicated functions. In the near future we plan to stop using get_subsystem and use the local state instead. Once we do so, I'll get rid of all the _safe functions except for the remove_namespace one which is still needed. Instead of calling the _safe functions from a wrapper I'll add the "with rpc_lock" statement to the beginning of each function.